### PR TITLE
Add url parameter to Twitter events

### PR DESCRIPTION
**Pull request by seblw**
Example log output:

INFO[1566] Event received: Twittertest / tweet - is triggered whenever someone you follow tweets 
INFO[1566] 	Options: {username string mcclure111}       
INFO[1566] 	Options: {text string RT @tweegeemee: 170213_063132_b.clj https://t.co/6XDuJuYtWQ #ProceduralArt #generative https://t.co/7kgwYxDg8G} 
INFO[1566] 	Options: {url string https://twitter.com/statuses/831266508819030019}

### CODE REVIEW

**Comment by seblw** [comment]
#70

File: bees/twitterbee/twitterbee.go [hunk 1]
```diff
@@ -171,6 +171,11 @@ func (mod *TwitterBee) handleStreamEvent(item interface{}) {
 					Type:  "string",
 					Value: status.Text,
 				},
+				{
+					Name:  "url",
+					Type:  "string",
+					Value: "https://twitter.com/statuses/" + status.IdStr,
+				},
```
Comment (muesli) [inline bees/twitterbee/twitterbee.go]: The same needs be done for the EventTweet case below.

File: bees/twitterbee/twitterbeefactory.go [hunk 2]
```diff
@@ -168,6 +168,11 @@ func (factory *TwitterBeeFactory) Events() []bees.EventDescriptor {
 					Description: "text content of the tweet",
 					Type:        "string",
 				},
+				{
+					Name:        "url",
+					Description: "URL of the tweet",
+					Type:        "string",
```
Comment (muesli) [inline bees/twitterbee/twitterbeefactory.go]: Use the type "url" for those url parameters, please. I know it's not consistently used across all bees yet, but I'm about to make that switch in the near future.

Comment (seblw) [inline bees/twitterbee/twitterbeefactory.go]: ok, should the name be changed to something like twitt_url to avoid confusion or it's ok as is?

Comment (muesli) [inline bees/twitterbee/twitterbeefactory.go]: The name 'url' is totally fine. It's namespaced to this Bee, anyway, and eventually the type shouldn't be a user-facing value, so I don't mind it being a name and type at the same time.

**Pull request merged**